### PR TITLE
wizard: add new Trezor Safe 5

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

**Comment by plowsof** [comment]
```

**Review by tobtoht — APPROVED** [review]

**Comment by plowsof** [comment]
Screenshot from 2024-08-08 20-24-26
:heavy_check_mark:

**Review by plowsof — APPROVED** [review]

**Review by tobtoht — APPROVED** [review]

File: qml.qrc [hunk 1]
```diff
@@ -283,6 +283,7 @@
         <file>images/ledgerNanoX.png</file>
         <file>images/ledgerStax.png</file>
         <file>images/trezor3.png</file>
+        <file>images/trezor5.png</file>
```
Comment (Rhahgan21) [inline qml.qrc:286]: Good

Comment (Rhahgan21) [inline qml.qrc:286]: Good

**Pull request merged**